src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedThermoElasticWellbore/Example.rst

```diff
@@ -0,0 +1,256 @@
+
+-----------------------------------------------------------
+Collecting output data
+-----------------------------------------------------------
+
+It is convenient to collect data in hdf5 format that can be easily post-processed using Python. To collect the temperature field in the three layers for all the time steps, the following XML blocks need to be defined:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_TemperatureCollection -->
+  :end-before: <!-- SPHINX_TemperatureCollectionEnd -->
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_TemperatureTimeHistory -->
+  :end-before: <!-- SPHINX_TemperatureTimeHistoryEnd -->
+
+Similarly, the following blocks are needed to collect the solid stress:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_SolidStressCollection -->
+  :end-before: <!-- SPHINX_SolidStressCollectionEnd -->
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_SolidStressTimeHistory -->
+  :end-before: <!-- SPHINX_SolidStressTimeHistoryEnd -->
+
+The displacement field can be collected for the whole domain using ``nodeManager`` as follows
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_DisplacementCollection -->
+  :end-before: <!-- SPHINX_DisplacementCollectionEnd -->
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_DisplacementTimeHistory -->
+  :end-before: <!-- SPHINX_DisplacementTimeHistoryEnd -->
+
+Also, periodic events are required to trigger the collection of this data on the mesh. For example, the periodic events for collecting the displacement field are defined as:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_benchmark.xml
+  :language: xml
+  :start-after: <!-- SPHINX_DisplacementPeriodicEvent -->
+  :end-before: <!-- SPHINX_DisplacementPeriodicEventEnd -->
+
+
+---------------------------------
+Results and benchmark
+---------------------------------
+
+A good agreement between the GEOSX results and analytical results for temperature distribution around the cased wellbore is shown in the figures below:
+
+.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedThermoElasticWellbore/thermoElastic_casedWellbore_temperature.py
+
+.. _resultsCasedThermoElasticWellbore_temperatureFig:
+.. figure:: temperature.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Temperature distribution around the cased wellbore
```

Comment:
Duplicated figure should be removed, which is generated with the python script.

Reply:
done